src/db/mysql/tables_and_columns.sql

```diff
@@ -1,11 +1,46 @@
 CREATE DATABASE skillsync;
 USE skillsync;
 
-CREATE TABLE users (
+CREATE TABLE applicants (
     id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Since the id is a uuid it should not be auto incremented it's a normal string in the end